main_tests/setup_env.sh

```diff
@@ -391,6 +442,16 @@ dhcp-range=192.168.0.100,192.168.0.150,24h
 	EOF
 }
 
+setup_sync_datetime() {
+	sudo_required
+
+	# Try to use NTP. May fail if there is no timesync service (e.g. systemd-timesyncd)
+	sudo timedatectl set-ntp true && return
+
+	# Alternatively, use a much less precise source, but still good enough for certificates to be valid
+	sudo date +"%d %b %Y %T %Z" -s "$(curl -s --head http://google.com | grep '^Date:' | cut -d' ' -f 3-)"
```

Comment:
just a nitpick but since this step and prereqs install require internet, maybe it would be useful to add a short connection check in the beginning of the script, to avoid going further if the device is not connected to the internet (maybe just a ping?)

Reply:
Does [def3e61](https://github.com/analogdevicesinc/production-tests/pull/20/commits/def3e6164623361f59a4c4dc7ed0f894984bc6c8) look alright?

Reply:
looks good to me